tests: Similar issue to #1

Just need to fix another 2 compiler warnings in the test suite.

```
../tests/t1000-add-xpath-node.c:47:9: warning: ignoring return value of ‘write’ declared with attribute ‘warn_unused_result’ [-Wunused-result]
   47 |         write(fd, template, sizeof(template) - 1);

```

--- a/tests/t1000-add-xpath-node.c
+++ b/tests/t1000-add-xpath-node.c
@@ -44,7 +44,9 @@ int main(int argc, char **argv)
 	int fd = mkstemp(in);
 	if (fd < 0)
 		exit(EXIT_FAILURE);
-	write(fd, template, sizeof(template) - 1);
+	int ret = write(fd, template, sizeof(template) - 1);
+	if (ret < 0)
+		exit(EXIT_FAILURE);
 
 	/* test 1 */
 	diag("add node <theme><cornerradius> using xpath (lowercase)");

--- a/tests/t1001-nodenames.c
+++ b/tests/t1001-nodenames.c
@@ -32,7 +32,9 @@ int main(int argc, char **argv)
 	int fd = mkstemp(in);
 	if (fd < 0)
 		exit(EXIT_FAILURE);
-	write(fd, template, sizeof(template) - 1);
+	int ret = write(fd, template, sizeof(template) - 1);
+	if (ret < 0)
+		exit(EXIT_FAILURE);
 
 	/* test 1 */
 	diag("generate simple xpath style nodename");

